tbx/core/models.py

```diff
@@ -103,60 +98,41 @@ class Meta:
         abstract = True
 
 
-# Home Page
-class HomePageFeaturedPost(Orderable):
-    page = ParentalKey(
-        "torchbox.HomePage", on_delete=models.CASCADE, related_name="featured_posts"
-    )
-    featured_post = models.ForeignKey(
-        "wagtailcore.Page",
+class HomePagePartnerLogo(Orderable):
+    page = ParentalKey("torchbox.HomePage", related_name="logos")
+    image = models.ForeignKey(
+        "images.CustomImage",
         null=True,
         blank=True,
         on_delete=models.SET_NULL,
         related_name="+",
     )
 
     panels = [
-        PageChooserPanel("featured_post", ["blog.BlogPage", "work.HistoricalWorkPage"]),
+        FieldPanel("image"),
     ]
 
 
-class HomePageHeroImage(Orderable):
-    page = ParentalKey(
-        "torchbox.HomePage", on_delete=models.CASCADE, related_name="hero_images"
-    )
-    image = models.ForeignKey(
-        "images.CustomImage",
-        help_text="The hero images will be displayed in a random order.",
-        null=True,
-        blank=False,
-        on_delete=models.CASCADE,
-        related_name="+",
-    )
-
-
+# Home Page
 class HomePage(ColourThemeMixin, ContactMixin, SocialFields, Page):
     template = "patterns/pages/home/home_page.html"
-    hero_intro_primary = models.TextField(blank=True)
-    hero_intro_secondary = models.TextField(blank=True)
-    intro_body = RichTextField(blank=True)
-    work_title = models.TextField(blank=True)
-    blog_title = models.TextField(blank=True)
-    clients_title = models.TextField(blank=True)
+    statement = models.TextField(blank=True)
+    introduction = models.TextField(blank=True)
+    body = StreamField(HomePageStoryBlock(), use_json_field=True)
 
     class Meta:
         verbose_name = "Homepage"
 
     content_panels = Page.content_panels + [
         MultiFieldPanel(
             [
-                FieldPanel("hero_intro_primary"),
-                FieldPanel("hero_intro_secondary"),
-                InlinePanel("hero_images", label="Hero Images", max_num=6, min_num=1),
+                FieldPanel("statement"),
+                FieldPanel("introduction"),
             ],
-            heading="Hero intro",
+            heading="Introductory block",
         ),
-        InlinePanel("featured_posts", label="Featured Posts", max_num=3),
+        InlinePanel("logos", label="Partner Logos", max_num=7),
```

Comment:
```suggestion
        InlinePanel("logos", heading="Partner logos", label="logo", max_num=7),
```

Then it will look like this

<details><summary>Illustration</summary>

![image](https://github.com/torchbox/torchbox.com/assets/7713776/ef51fc88-0c49-40e3-a396-3062f7304e6e)

</details>